<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>637</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1196</width>
    <height>637</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QQuickWidget" name="quickWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>781</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#A52A2A; 
border-left-color: 3px solid rgb(85, 0, 0);</string>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>fIrst Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>145</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Tablet login-pana.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>100</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Last name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Faculty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>160</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Phpne Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>310</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>170</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>100</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Middle Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>240</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="placeholderText">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  border: none;
  border-bottom: 2.5px solid #A52A2A;  /* Deep brown color code */
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>20</width>
     <height>461</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>571</width>
     <height>141</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{

border:none;
border-bottom: 2px solid brown;
}</string>
   </property>
   <property name="placeholderText">
    <string>Tell us About Yourself</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 9pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>...Welcome...</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
